app/questionnaire/routing/operator.py

```diff
@@ -50,3 +53,23 @@ def evaluate(
     @staticmethod
     def _any_operands_none(*operands: Union[Sequence, ValueTypes]) -> bool:
         return any(operand is None for operand in operands)
+
+
+OPERATION_MAPPING: dict[str, str] = {
```

Comment:
I can live with it, but I as you probably had guess I wasn't a huge fan of mapping the operation to the function name with a string, it felt to me more likely to break, but did add more padding.

Reply:
Understandable, I mainly changed it so rules evaluator is not responsible for the operation. The str mapping is a consequence of our operator names, as we can't have a 1:1 method name like placeholder transforms. However, we could still do what placeholder does, but just make a mapping for the operators that are "special" i.e `!=` etc. Don't think it's worth changes the operator name itself for the sake of python.